feature.js

```diff
@@ -100,6 +100,8 @@
       return !!test;
     })(),
 
+    Animation : !!HTMLElement.prototype.animate,
```

Comment:
I'm curious about your thoughts of changing this to `('animate' in util.create('div'))`, or something too the effect, at least for consistency's sake. And should `Animation` change to just `animate`?